eng/bomgenerator/pom.xml

```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
-      <version>1.7.32</version>  <!-- {x-version-update;org.slf4j:slf4j-log4j12;external_dependency} -->
+      <version>3.5.4</version> <!-- {x-version-update;org.apache.maven:maven-core;external_dependency} -->
```

Comment:
This looks wrong, we should not change the version in such a drastic way. I think it's actually taking the one from `maven-core` instead of `slf4j`.
```suggestion
      <version>1.7.32</version> <!-- {x-version-update;org.slf4j:slf4j-log4j12;external_dependency} -->
```

Reply:
That said, since this is under the bomgenerator folder I don't think it will be relevant for the ACS PhoneNumbers release, so you could technically ignore it as long as everything is done from the release branch.

Reply:
Ack. Still not sure why the script bade this change.